features/step_definitions/content.rb

```diff
@@ -0,0 +1,8 @@
+Then(/^it should show the names of the participating teams$/) do
+  expect(page).to have_text(@matches.last.team_home.name)
```

Comment:
06_Testing.pdf slide 33: "Do not test for existence of actual text" - Does this mean you should rather use have_content than have_text?

Reply:
That actually referred to not testing text because it might be translated.

This tests whether the data from the model can be found on the page, which should be fine.